### PR TITLE
Add CSI work diary module

### PR DESCRIPTION
**Pull request by Rick00Kim**
Spec
1. Python 3.8.2
2. Main libraries
    - Flash framework
    - Selenium
    - Slack client
3. Deploy Server
    - Heroku(No using database)

### CODE REVIEW

File: csi_work_diary/app/message_format/app_view.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    "type": "plain_text",
+                    "text": "Add specific date",
+                    "emoji": true
+                },
+                "value": "click_me",
+                "action_id": "addSpecificDate"
+            }
+        }
+    ]
+}
```
Comment (Rick00-Helper) [inline csi_work_diary/app/message_format/app_view.json:71]: EOF

File: csi_work_diary/app/message_format/hotel_list.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
+[
```
Comment (Rick00-Helper) [inline csi_work_diary/app/message_format/hotel_list.json:1]: Never use

File: csi_work_diary/app/message_format/new_user.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                "action_id": "start_time"
+            },
+            "label": {
+                "type": "plain_text",
+                "text": "Start Time",
+                "emoji": true
+            }
+        }
+    ]
+}
```
Comment (Rick00-Helper) [inline csi_work_diary/app/message_format/new_user.json:91]: EOF

**Review by Rick00-Helper — CHANGES_REQUESTED** [review]
Please check some code

**Comment by Rick00Kim** [comment]
This pull request will merge after complete to-do.

File: csi_work_diary/app/message_format/hotel_list.json (hunk 2 above)
Comment (Rick00Kim) [reply csi_work_diary/app/message_format/hotel_list.json]: #2 Resolved

File: csi_work_diary/app/message_format/app_view.json (hunk 1 above)
Comment (Rick00Kim) [reply csi_work_diary/app/message_format/app_view.json]: 49f6363 resolved

File: csi_work_diary/app/message_format/new_user.json (hunk 3 above)
Comment (Rick00Kim) [reply csi_work_diary/app/message_format/new_user.json]: 49f6363 resolved

**Review by Rick00-Helper — APPROVED** [review]
LGTM

**Comment by Rick00Kim** [comment]
@Rick00-Helper 
Thank you for check.
I will merge this pull request.

**Pull request merged**